channels/unsorted.m3u

```diff
@@ -422,103 +344,79 @@ https://cdn2.zencast.tv:30443/misiones4/live/chunks.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="https://ovacion.pe/sites/default/files/logo-ovacion.png" group-title="",Ovacion
 http://cdn2.ujjina.com:1935/iptvovacion1/liveovacion1tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="" group-title="",Cultura 24 (Opt-2)
-http://vs8.live.opencaster.com/cultura24/_definst_/mp4:cultura24_480/chunklist_w1999901240.m3u8
+https://vs8.live.opencaster.com/cultura24/_definst_/mp4:cultura24_480/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="" group-title="",Turbo Mix Radio TV
 http://167.114.116.212:1935/tvmb/turbomix/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="http://i.imgur.com/fzsER.png" group-title="",NTN24
 https://rcnhd.cdn.net.co/live04/ntn24_480p.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Spanish" tvg-logo="http://www.suram.com.co/wp-content/uploads/2017/04/LOGO-SURAM.png" group-title="",Suram TV
-https://5b3050bb1b2d8.streamlock.net/suramtv/suramtv/chunklist_w1810740705.m3u8
```

Comment:
This exists in https://github.com/iptv-org/iptv/blob/d1d51f0f2283836bc70ff7578d9344c4f4ef4e53/channels/co.m3u#L27 which is why it was removed.

I converted the links to adaptive and then ran "npm run format"